components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/client-async.ts

```diff
@@ -35,9 +45,9 @@ export class HttpAsyncClient {
 		}
 	};
 
-	postAsync(url, config, options) {
-		const newUrl = buildUrl(url, options);
-		const callback = createHttpResponseCallback(
+	public static postAsync(url: string, config: Config, options: Option): void {
```

Comment:
```ts
postAsync(url, config, options) {
...
```

was not a `static` method, do we want to make it static?

Otherwise the change should be as follows:

```ts
public postAsync(url: string, config: Config, options: Option): void {
...
```